INPUT_SCHEMA.json

```diff
@@ -19,12 +20,13 @@
       "editor": "hidden"
     },
     "lastReviewDate": {
-      "title": "Reviews since date",
+      "title": "Reviews since date (YYYY-MM-DD)",
       "type": "string",
       "description": "It is a date where the scrapper stops loading older reviews ",
       "default": "2019-01-01",
       "prefill": "2019-01-01",
-      "editor": "textfield"
+      "editor": "textfield",
+      "patternValue": "\\d{4}-\\d{2}-\\d{2}"
```

Comment:
can't build, use `apify vis` for testing the schema. the property name is called `pattern` not `patternValue`